src/argilla/client/feedback/dataset/mixins.py

```diff
@@ -0,0 +1,296 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import warnings
+from typing import TYPE_CHECKING, Optional, Type, Union
+from uuid import UUID
+
+from tqdm import trange
+
+from argilla.client.api import ArgillaSingleton
+from argilla.client.feedback.constants import PUSHING_BATCH_SIZE
+from argilla.client.feedback.dataset.remote import _ArgillaFeedbackDataset
+from argilla.client.feedback.schemas import (
+    LabelQuestion,
+    MultiLabelQuestion,
+    RankingQuestion,
+    RatingQuestion,
+    TextField,
+    TextQuestion,
+)
+from argilla.client.feedback.types import AllowedQuestionTypes
+from argilla.client.feedback.utils import feedback_dataset_in_argilla
+from argilla.client.sdk.v1.datasets import api as datasets_api_v1
+from argilla.client.workspaces import Workspace
+
+if TYPE_CHECKING:
+    import httpx
+
+    from argilla.client.client import Argilla as ArgillaClient
+    from argilla.client.feedback.dataset.local import FeedbackDataset
+    from argilla.client.sdk.v1.datasets.models import FeedbackDatasetModel
+
+warnings.simplefilter("always", DeprecationWarning)
+
+
+class ArgillaToFromMixin:
+    def push_to_argilla(
+        self: "FeedbackDataset",
+        name: Optional[str] = None,
+        workspace: Optional[Union[str, Workspace]] = None,
+        show_progress: bool = False,
+    ) -> _ArgillaFeedbackDataset:
+        """Pushes the `FeedbackDataset` to Argilla. If the dataset has been previously pushed to Argilla, it will be updated
+        with the new records.
```

Comment:
The second part of the message is outdated, right?
```suggestion
        """Pushes the `FeedbackDataset` to Argilla. 
```